src/pubsub/mod_pubsub.erl

```diff
@@ -421,32 +421,32 @@ init_plugins(Host, ServerHost, Opts) ->
     TreePlugin = tree(Host, gen_mod:get_opt(nodetree, Opts,
                                             fun(A) when is_binary(A) -> A end,
                                             ?STDTREE)),
-    ?DEBUG("** tree plugin is ~p", [TreePlugin]),
+    ?LOG_DEBUG(#{what => pubsub_tree_plugin, tree_plugin => TreePlugin}),
     gen_pubsub_nodetree:init(TreePlugin, Host, ServerHost, Opts),
     Plugins = gen_mod:get_opt(plugins, Opts,
                               fun(A) when is_list(A) -> A end, [?STDNODE]),
     PepMapping = gen_mod:get_opt(pep_mapping, Opts,
                                  fun(A) when is_list(A) -> A end, []),
-    ?DEBUG("** PEP Mapping : ~p~n", [PepMapping]),
+    ?LOG_DEBUG(#{what => pubsub_pep_mapping, pep_mapping => PepMapping}),
     PluginsOK = lists:foldl(pa:bind(fun init_plugin/5, Host, ServerHost, Opts), [], Plugins),
     {lists:reverse(PluginsOK), TreePlugin, PepMapping}.
 
 init_plugin(Host, ServerHost, Opts, Name, Acc) ->
     Plugin = plugin(Host, Name),
     case catch apply(Plugin, init, [Host, ServerHost, Opts]) of
         {'EXIT', _Error} ->
-            ?ERROR_MSG("event=pubsub_plugin_init_failed,plugin=~p,host=~s,pubsub_host=~s,opts=~p",
-                       [Plugin, ServerHost, Host, Opts]),
+            ?LOG_ERROR(#{what => pubsub_plugin_init_failed, plugin => Plugin,
+                host => ServerHost, pubsub_host => Host, opts => Opts}),
```

Comment:
So, `server => ServerHost, sub_host => Host`. And the same format everywhere :) (it's also described in my fields docs)